tests/test_core.py

```diff
@@ -938,6 +940,101 @@ def test_scaper_instantiate():
         _compare_scaper_jams(jam, regjam)
 
 
+def create_scaper_with_random_seed(seed):
+    sc = scaper.Scaper(10.0, fg_path=FG_PATH, bg_path=BG_PATH, random_state=deepcopy(seed))
+    sc.ref_db = -50
+    sc.sr = 44100
+
+    # background
+    sc.add_background(
+        label=('const', 'park'),
+        source_file=(
+            'const',
+            'tests/data/audio/background/park/'
+            '268903__yonts__city-park-tel-aviv-israel.wav'),
+        source_time=('const', 0))
+
+    # foreground events
+    sc.add_event(
+        label=('const', 'siren'),
+        source_file=('const',
+                     'tests/data/audio/foreground/'
+                     'siren/69-Siren-1.wav'),
+        source_time=('uniform', 0, 8),
+        event_time=('truncnorm', 4, 1, 0, 8),
+        event_duration=('normal', 4, 1),
+        snr=('const', 5),
+        pitch_shift=None,
+        time_stretch=None)
+
+    sc.add_event(
+        label=('const', 'car_horn'),
+        source_file=('const',
+                     'tests/data/audio/foreground/'
+                     'car_horn/17-CAR-Rolls-Royce-Horn.wav'),
+        source_time=('uniform', 0, 8),
+        event_time=('truncnorm', 4, 1, 0, 8),
+        event_duration=('normal', 4, 1),
+        snr=('const', 20),
+        pitch_shift=None,
+        time_stretch=None)
+
+    sc.add_event(
+        label=('const', 'human_voice'),
+        source_file=('const',
+                     'tests/data/audio/foreground/'
+                     'human_voice/42-Human-Vocal-Voice-taxi-2_edit.wav'),
+        source_time=('const', 0),
+        event_time=('const', 7),
+        event_duration=('const', 2),
+        snr=('const', 10),
+        pitch_shift=None,
+        time_stretch=None)
+
+    return sc
+
+
+def test_generate_with_seeding(atol=1e-4, rtol=1e-8):
```

Comment:
This is awesome! But... can you add tests to compare the JAMS and text files as well? [This code](https://github.com/justinsalamon/scaper/blob/master/tests/test_core.py#L1141-L1173) does exactly this, so it's just a matter of incorporating it into this test.